shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -228,11 +218,8 @@ class FlutterTizenEngine : public TizenRenderer::Delegate {
   // AOT data for this engine instance, if applicable.
   UniqueAotDataPtr aot_data_;
 
-  // An event dispatcher for Ecore key events.
-  std::unique_ptr<KeyEventHandler> key_event_handler_;
-
-  // An event dispatcher for Ecore mouse events.
-  std::unique_ptr<TouchEventHandler> touch_event_handler_;
```

Comment:
The event handlers are not engine dependent now
I think It is better to depend on something that can be hosts to a flutter-tizen, such as a Window or any UI toolkit component.